SDVModTest/Tools.cs

```diff
@@ -1,106 +1,107 @@
-﻿using StardewValley;

```

Comment:
This file just has me confused. It looks like the only thing changed was to add a few usings, which shouldn't have been needed anyway.